paasta_tools/cli/cmds/status.py

```diff
@@ -1331,24 +1342,35 @@ def get_replica_state(pod: KubernetesPodV2) -> ReplicaState:
             # TODO: Take sidecar containers into account
             #   This logic likely needs refining
             state = ReplicaState.UNREACHABLE
-            main_container = [
-                c
-                for c in pod.containers
-                if c.name not in kubernetes_tools.SIDECAR_CONTAINER_NAMES
-            ]
+            main_container = next(
+                (
+                    c
+                    for c in pod.containers
+                    if c.name not in kubernetes_tools.SIDECAR_CONTAINER_NAMES
+                ),
+                None,
+            )
             if main_container:
```

Comment:
I do think it might be feasible that there is a short period of time in Running phase but the main container isn't up yet; regardless I think keeping this on UNKNOWN and then having a tip for UNKNOWN pods to just try reruning `status` (would catch this short inbetween period) and if they continue to see UNKNOWN, reach out to #paasta?

Reply:
Yeah fair point about it being a possible transitional state.  I guess let's talk about this later when we chat with qui